manifests/params.pp

```diff
@@ -6,6 +6,7 @@
   $purge_conf_dir         = false
   $restart                = false
   $root_password          = 'UNSET'
+  $install_secret_file    = '/.mysql_secret'
```

Comment:
When Puppet installs the MySQL-server package this file seems to end up at /.mysql_secret